great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -98,35 +109,40 @@ def __init__(
-            identifying unexpected values as judged by the upper- and lower- quantiles of the observed metric data.
+                identifying unexpected values as judged by the upper- and lower- quantiles of the observed metric data.
             quantile_statistic_interpolation_method: Applicable only for the "bootstrap" sampling method --
-            supplies value of (interpolation) "method" to "np.quantile()" statistic, used for confidence intervals.
-            estimator: choice of the estimation algorithm: "oneshot" (one observation) or "bootstrap" (default)
-            num_bootstrap_samples: Applicable only for the "bootstrap" sampling method -- if omitted (default), then
-            9999 is used (default in "https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.bootstrap.html").
-            bootstrap_random_seed: Applicable only for the "bootstrap" sampling method -- if omitted (default), then
-            uses "np.random.choice"; otherwise, utilizes "np.random.Generator(np.random.PCG64(bootstrap_random_seed))".
-            include_bootstrap_samples_histogram_in_details: Applicable only for the "bootstrap" sampling method -- if
-            True, then add 10-bin histogram of bootstraps to "details"; otherwise, omit this information (default).
+                supplies value of (interpolation) "method" to "np.quantile()" statistic, used for confidence intervals.
+            estimator: choice of the estimation algorithm: "oneshot" (one observation), "bootstrap" (default),
+                or "kde" (kernel density estimation).
+            n_resamples: Applicable only for the "bootstrap" and "kde" sampling methods -- if omitted (default), then
+                9999 is used (default in
+                "https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.bootstrap.html").
+            bw_method: Applicable only for the "kde" sampling method -- if omitted (default), then "scott" is used.
+                Possible values for the estimator bandwidth method are described at:
+                https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.gaussian_kde.html
+            random_seed: Applicable only for the "bootstrap" and "kde" sampling methods -- if omitted (default), then
+                uses "np.random.choice"; otherwise, utilizes "np.random.Generator(np.random.PCG64(random_seed))".
+            include_estimator_samples_histogram_in_details: Applicable only for the "bootstrap" sampling method -- if
+                True, then add 10-bin histogram of bootstraps to "details"; otherwise, omit this information (default).
             truncate_values: user-configured directive for whether or not to allow the computed parameter values
-            (i.e., lower_bound, upper_bound) to take on values outside the specified bounds when packaged on output.
+                (i.e., lower_bound, upper_bound) to take on values outside the specified bounds when packaged on output.
```

Comment:
@NathanFarmer Is this just the indentation?

Reply:
@alexsherstinsky I find it very difficult to read without the indentation.